DSharpPlus/Entities/DiscordChannel.cs

```diff
@@ -90,6 +102,21 @@ public class DiscordChannel : SnowflakeObject, IEquatable<DiscordChannel>
         /// </summary>
         [JsonIgnore]
         public string Mention => Formatter.Mention(this);
+        
+        /// <summary>
+        /// Gets this channel's children. This applies only to channel categories.
+        /// </summary>
+        [JsonIgnore]
+        public IEnumerable<DiscordChannel> Children {
+            get
+            {
+                if (!IsCategory)
+                {
+                    throw new ArgumentException("Only channel categories contain children");
+                }
+                return Guild.Channels.Where(e => e.ParentId == Id);
```

Comment:
Here it's probably a better idea to use `_guilds` member, although that's a minor opt. Also braces.